### ISSUE TITLE
2020-12-20

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#399]
411Check my photo; done; (2020-12-10)
412What is python \? 【python】代码换行的几种方法; 对于一般表达式来说，反斜杠后直接回车即可实现续行，使用的关键在于反斜杠后不能用空格或者其他符号; 三引号; (2020-12-10)
413Try to read def train; what is tf.nn.l2_normalize? Normalizes along dimension axis using an L2 norm; (2020-12-10)
414What is tf.reduce_sum? Computes the sum of elements across dimensions of a tensor; (2020-12-10)
415What is tf.sqrt? Computes element-wise square root of the input tensor; (2020-12-10)
416What is tf.nn.embedding_lookup? Look up; 仰望；查阅；尊敬；拜访；改善; Looks up embeddings for the given ids from a list of tensors; embedding_lookup function retrieves rows of the params tensor. The behavior is similar to using indexing with arrays in numpy; (2020-12-10)
417What is tf.abs? Computes the absolute value of a tensor; (2020-12-10)
418What is tf.placeholder? So far we have used Variables to manage our data, but there is a more basic structure, the placeholder. A placeholder is simply a variable that we will assign data to at a later date. It allows us to create our operations and build our computation graph, without needing the data. In TensorFlowterminology, we then feed data into the graph through these placeholders; (2020-12-10)
419What is tf.train.GradientDescentOptimizer? Optimizer that implements the gradient descent algorithm; (2020-12-10)
